PINT incorrectly demands POSEPOCH when proper motion not set

With the par file
```
PSR J1234+5678
ELAT 0
ELONG 0
F0 1
DM 10
```
PINT raises MissingParameter with the message:
```
E               pint.models.timing_model.MissingParameter: Astrometry.POSEPOCH
E                 POSEPOCH or PEPOCH are required if PM is set.
```

--- a/tests/test_model.py
+++ b/tests/test_model.py
@@ -1,23 +1,24 @@
 #! /usr/bin/env python
+import io
 import os
 import time
-from io import StringIO
+
 import astropy.units as u
 import matplotlib.pyplot as plt
 import numpy as np
 from astropy import log
+from pinttestdata import datadir
 
 import pint.models as tm
 from pint import toa
 from pint.residuals import Residuals
-from pinttestdata import datadir
 
 
 def test_simple_parfile():
     """Test a simple parfile without frequency derivatives, proper motion,
     PEPOCH and POSEPOCH."""
     model = tm.get_model(
-        StringIO(
+        io.StringIO(
             """
             PSR J1234+5678
             ELAT 0

--- a/tests/test_model.py
+++ b/tests/test_model.py
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 import os
 import time
-
+from io import StringIO
 import astropy.units as u
 import matplotlib.pyplot as plt
 import numpy as np
@@ -11,6 +11,25 @@ import pint.models as tm
 from pint import toa
 from pint.residuals import Residuals
 from pinttestdata import datadir
+
+
+def test_simple_parfile():
+    """Test a simple parfile without frequency derivatives, proper motion,
+    PEPOCH and POSEPOCH."""
+    model = tm.get_model(
+        StringIO(
+            """
+            PSR J1234+5678
+            ELAT 0
+            ELONG 0
+            F0 1
+            DM 10
+            """
+        )
+    )
+
+    assert model.ELAT.value == model.ELONG.value == 0
+    assert model.PEPOCH.quantity is None and model.POSEPOCH.quantity is None
 
 
 def test_model():
@@ -56,8 +75,9 @@ def test_model():
     #                                     ['tt2tb', 'roemer', 'post_phase',
     #                                      'shapiro', 'shapiroJ'])
     tempo2_vals = np.genfromtxt(
-        parfile + ".tempo2_test", names=True, comments="#", dtype=np.longdouble
+        f"{parfile}.tempo2_test", names=True, comments="#", dtype=np.longdouble
     )
+
     t2_resids = tempo2_vals["post_phase"] / float(m.F0.value) * 1e6 * u.us
     diff_t2 = (resids_us - t2_resids).to(u.ns)
     diff_t2 -= diff_t2.mean()
@@ -75,8 +95,9 @@ def test_model():
 
         log.info("Running TEMPO1...")
         t1_result = np.genfromtxt(
-            t1_parfile + ".tempo_test", names=True, comments="#", dtype=np.longdouble
+            f"{t1_parfile}.tempo_test", names=True, comments="#", dtype=np.longdouble
         )
+
         t1_resids = t1_result["residuals_phase"] / float(m.F0.value) * 1e6 * u.us
         did_tempo1 = True
         diff_t1 = (resids_us - t1_resids).to(u.ns)

